packages/fxa-shared/subscriptions/configuration/product.ts

```diff
@@ -38,6 +38,14 @@ const buildProductConfigSchema = (baseSchema: joi.ObjectSchema) =>
       'urls'
     );
 
+export type ProductConfigUrlConfig = Partial<UrlConfig> & {
+  download: string;
+  privacyNotice: string;
+  termsOfService: string;
+  termsOfServiceDownload: string;
+  webIcon: string;
+};
+
```

Comment:
Can you leave a comment on why we need this please? 🙇

Reply:
In `ProductConfig.urls`, these fields are required, but since we only make them required after initial definition, if we kept the type for `ProductConfig.urls` as `UrlConfig` the type for `ProductConfig.urls` would be incorrect, and still shows all these fields as optional.

And if you meant a comment in the code, I can do that as well. 😅

Reply:
🤦 I'm sorry, I did mean a code comment.